clipper_admin/clipper_manager.py

```diff
@@ -735,6 +744,105 @@ def centered_predict(inputs):
 
         return deploy_result
 
+    def _register_app_and_check_success(self, name, input_type, default_output,
+                                        slo_micros):
+        if self.register_application(name, name, input_type, default_output,
+                                     slo_micros):
+            print("Application registration sucessful! Deploying model.")
+            return True
+        print("Application registration unsuccessful. Will not deploy model.")
+        return False
+
+    def activate_predict_function(self,
+                                  name,
+                                  predict_function,
+                                  default_output,
+                                  input_type,
+                                  model_version=DEFAULT_MODEL_VERSION,
+                                  slo_micros=DEFAULT_SLO_MICROS,
+                                  labels=DEFAULT_LABEL,
+                                  num_containers=1):
+        """Registers an app and deploys provided predict function as a model.
+
+        Parameters
+        ----------
+        name : str
+            The to be assigned to the registered app and deployed model.
+        predict_function : function
```

Comment:
Don't say "refer to documentation in ..." That is really confusing and leads to hard to read docs. Just copy the text.

As we refactor this documentation, we'll create some module-level documentation about deploying models and the meaning of these fields.